pnpm-lock.yaml

```diff
@@ -245,19 +245,19 @@ importers:
         version: 4.0.4
       '@dnd-kit/core':
         specifier: ^6.1.0
-        version: 6.1.0(react-dom@18.2.0)(react@18.2.0)
+        version: 6.1.0(react-dom@18.2.0(react@18.2.0))(react@18.2.0)
       '@dnd-kit/modifiers':
         specifier: ^7.0.0
-        version: 7.0.0(@dnd-kit/core@6.1.0)(react@18.2.0)
+        version: 7.0.0(@dnd-kit/core@6.1.0(react-dom@18.2.0(react@18.2.0))(react@18.2.0))(react@18.2.0)
       '@dnd-kit/sortable':
         specifier: ^8.0.0
-        version: 8.0.0(@dnd-kit/core@6.1.0)(react@18.2.0)
+        version: 8.0.0(@dnd-kit/core@6.1.0(react-dom@18.2.0(react@18.2.0))(react@18.2.0))(react@18.2.0)
       '@frontify/app-bridge':
         specifier: ^3.0.0 || ^4.0.0-alpha.0
-        version: link:../app-bridge
```

Comment:
This here is off :D

Reply:
Interesting, but `pnpm i` generates my version.

Reply:
Most likely you did `pnpm add @frontify/app-bridge@workspace:^ --filter {packages/guideline-blocks-settings}` to build and test etc

Reply:
Not 100% sure just feels odd, but yeah I don't think it matters